packages/nx-plugin/src/generators/app/lib/create-project.ts

```diff
@@ -0,0 +1,150 @@
+import { NormalizedOptions } from '../generator';
+import type { ProjectConfiguration, Tree } from '@nrwl/devkit';
+import { addProjectConfiguration } from '@nrwl/devkit';
+
+export function createProject(
+  tree: Tree,
+  normalizedOptions: NormalizedOptions
```

Comment:
Can we only pass in the properties we need from normalized options. This makes it easier to identify what this function actually depends on. We really only need: `projectRoot`,  `projectName`, and `parsedTags`